src/features/metadata-based-view/__tests__/MetadataBasedItemList.test.js

```diff
@@ -17,16 +17,44 @@ describe('features/metadata-based-view/MetadataBasedItemList', () => {
     const currentCollection = {
         items: [
             {
+                id: '1',
                 metadata: {
-                    data: { type: 'bill', amount: 500 },
+                    id: '11',
+                    fields: [
+                        {
+                            name: 'type',
+                            type: 'string',
+                            value: 'bill',
+                        },
+                        {
+                            name: 'amount',
+                            type: 'float',
+                            value: 100,
+                        },
+                    ],
                 },
                 name: 'name1.pdf',
+                size: '123',
             },
             {
+                id: '2',
                 metadata: {
-                    data: { type: 'receipt', amount: 200 },
+                    id: '22',
+                    fields: [
+                        {
+                            name: 'type',
+                            type: 'string',
+                            value: 'receipt',
+                        },
+                        {
+                            name: 'amount',
+                            type: 'float',
+                            value: 200,
```

Comment:
Since the type of the value is `float`, should we test with a float instead of an integer?